### PR TITLE
Issue #6586: aligned javadoc/xdoc for DefaultComesLast

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6586 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java [hunk 1]
```diff
@@ -28,20 +28,56 @@
 
 /**
  * <p>
- * Check that the {@code default} is after all the {@code case}s
- * in a {@code switch} statement.
+ * Check that the {@code default} is after all the {@code case} in a {@code switch} statement.
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java]: Why remove the `s` from `{@code case}s`?

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java]: `XdocsJavaDocsTest` fails with
```
Expected:
all the {@code case} s
Actual: 
all the {@code case}s
```
Do you know where the extra space might come from?

Comment (rnveach) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java]: Yes.
https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java#L366-L380

Basically the tool just sees the visible text and not the whitespaces. So it has to use it's brain to estimate where a space should be and should not be.
In this case, it thinks `s` is another word and is like "There should be a space splitting these too apart".

Comment (romani) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java]: Let's move this to separate issue, and proceed with the javadoc sync

**Comment by pbludov** [comment]
@rnveach
Done. `<code>case</code>s`  replaced with `cases`.

**Comment by rnveach** [comment]
@pbludov Are you going to try to fix it so we can have plurals next to code tags?

**Comment by pbludov** [comment]
> @pbludov Are you going to try to fix it so we can have plurals next to code tags?

No. It looks weird:
https://checkstyle.org/config_coding.html#DefaultComesLast_Description

On my system it rendered almost as "caseS". It is better to avoid such situations.

**Pull request merged**